hw/hw1/ex_01.py

```diff
@@ -0,0 +1,10 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:* 
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+print("Найду сумму цифр трехзначного числа. введи: ")
+x = str(input());
```

Comment:
Можно сделать так.
```suggestion
x = input("Найду сумму цифр трехзначного числа. введи: ")
```
А ещё в конце строки не нужно ставить `;` . И функции `input` всегда возвращает строковый тип данных.